### ISSUE TITLE
Feature Request: Multiple Alignments

### ISSUE THREAD

**Issue by EarthCitizen** [tuvistavie/atom-align-regexp#5]
If I have the following text:

```
a|   a   |y
some| thing| else
```
It would be great to be able to align by `|` multiple times in the same command execution and end up with this:
```
a    | a     | y
some | thing | else
```

**Comment by danhper** [comment]
Thanks for the idea :smile: 
Just published 0.4 with this feature, see the README for instructions.

**Comment by EarthCitizen** [comment]
That was a fast implementation! Thanks to you too!
